src/polytopes/chain.jl

```diff
@@ -4,59 +4,31 @@
 
 """
     Chain(p1, p2, ..., pn)
+    Chain(points::AbstractVector)
```

Comment:
```suggestion
```

We can review the signatures of all polytopes in a separate PR 👍🏽

Reply:
I’ve made the changes to docstrings more minimal in the newer version.